lib/buffer.js

```diff
@@ -295,6 +295,17 @@ function byteLength(string, encoding) {
 Buffer.byteLength = byteLength;
 
 
+Object.defineProperty(Buffer.prototype, 'dataView', {
+  get: function() {
```

Comment:
You can't set something that only has a getter. The current body of the getter is fine.